docker/docker_transport_test.go

```diff
@@ -163,6 +163,20 @@ func TestReferenceNewImage(t *testing.T) {
 	defer img.Close()
 }
 
+func TestReferenceNewImageWithInvalidRegistryToken(t *testing.T) {
+	ref, err := ParseReference("//busybox")
+	require.NoError(t, err)
+	_, err = ref.NewImage(context.Background(), &types.SystemContext{
+		RegistriesDirPath:        "/this/doesnt/exist",
+		DockerPerHostCertDirPath: "/this/doesnt/exist",
+		ArchitectureChoice:       "amd64",
+		OSChoice:                 "linux",
+		DockerRegistryToken:      "invalid",
+	})
+	// This should fail due to invalid registry token
+	require.Error(t, err)
+}
```

Comment:
(Non-blocking: I’m not quite sure this is worth testing here.)

Reply:
I removed
https://github.com/containers/image/pull/842/commits/1ffed6145a11a60c364cf85a1c234d4f8e32c292